data/block/block.proto

```diff
@@ -59,6 +59,19 @@ message PeerChange {
   uint32 ShardIdDest = 2 [(gogoproto.jsontag) = "shardIdDest"];
 }
 
+// ExecutionResult defines the dto that will hold information about execution result
+message ExecutionResult {
+  bytes  HeaderHash                         = 1 [(gogoproto.jsontag) = "headerHash"];
+  uint64 Nonce                              = 2 [(gogoproto.jsontag) = "headerNonce"];
```

Comment:
rename json tag to nonce? to match the field name, or rename the field to HeaderNonce